test/mp_unit/proxy_channel_tests.cu

```diff
@@ -242,14 +242,14 @@ __global__ void kernelProxyLLPingPong(int* buff, mscclpp::LLPacket* putPktBuf, m
       if (CheckCorrectness) {
         // If each thread reads 8 bytes at once, we don't need a barrier after getPackets().
         // __syncthreads();
-        for (int j = threadId; j < nPkt; j += numThreads) {
-          if (buffPtr[2 * j] != getOffset + i + 2 * j) {
+        for (size_t j = threadId; j < nPkt; j += numThreads) {
+          if (buffPtr[2 * j] != getOffset + i + 2 * (int)j) {
```

Comment:
j is size_t, but here is int. It's strange.

Reply:
Thanks for the comment, I fixed this.